src/main/java/edu/kit/compiler/transform/TypeMapper.java

```diff
@@ -96,6 +96,17 @@ public Type getDataType(DataType type) {
         };
     }
 
+    /**
+     * Returns the corresponding Firm `PointerType` for the given class.
+     * 
+     * @param name the class name
+     * @return the corresponding Firm type
+     */
+    public Type getClassPointerType(int name) {
```

Comment:
This should probably be a method of `ClassEntry`, i.e. `ClassEntry::getPointerType()`.